common/flags/Flags.java

```diff
@@ -100,7 +100,15 @@ static String getFlagValue(String name) {
     if (!flagData.getHasValue()) {
       return null;
     }
-    return instance().getFlag(name).getValue();
+    if (flagData.getIsListFlag()){
+      return flagData.getValue()
+          // deleting characters "[" and "]" at the beginning and end of the List and spaces after commas
+          // for correct parse values
+          .replace("[", "")
```

Comment:
Didn't you remove this in the previous PR in `getDefaultValue`?

Reply:
Yes, I first deleted it. But after we decided not to support the commas at `strings list flag`, I decided that it would be better to do this in one place than to override `getDefault()` method for each list flag.

Reply:
I think it's better to store it correctly in the first place, than to remember to correct it every time the data is read - or maybe there's some issue with correcting it at `getDefault()`?